### DISCUSSION TITLE
Ero truck simulator 2 server error

### DISCUSSION THREAD

**Discussion by Floki1996-dot** [GameServerManagers/LinuxGSM#4477] [Q&A]
Hi i was trying to start ETS2 server on linux VPS Ubuntu 22.04 and i get this error in log's like something is missing...

[MP] =========================================================================
*** ERROR *** : [MP] Error: Server packages file not found. This is required to load proper map, DLCs and mods.
*** ERROR *** : [MP] Error: Use "export_server_packages" command in game to export server game settings.
*** ERROR *** : [MP] Error: See server_readme.txt.
*** ERROR *** : [MP] Error: Server was teminated.

anyone have the same problem?

**Comment by batboyboy1990** [comment]
metoo!

**Reply by h3o66** [reply]
do you have the same error or a different ?
please check the logfile `log/console/ets2server-console.log` in the lgsm path

**Reply by batboyboy1990** [reply]
yes , and I have "Euro Truck Simulator 2 [mp] Error: Failed to export server packages info (/home/server_packages.sii)" if export server packages  :(

**Comment by dgibbs64** [comment]
I think there are some docs on the about importing a file from the client. Found this might help https://steamcommunity.com/sharedfiles/filedetails/?id=2900682260 

Its a bit of a pain

**Reply by batboyboy1990** [reply]
Ok thanks im doit works!

**Comment by stalker-qa** [comment]
Do you know how to open ports? I created rules in Ubuntu but the ports are not open. But the server is online. Did you open the ports?
